runtime/memory/cortex_m_wrapping.c

```diff
@@ -27,6 +27,19 @@ void expand_memory() {
     memory_size += WASM_PAGE_SIZE;
 }
 
+i32 instruction_memory_size() {
+    return memory_size / WASM_PAGE_SIZE;
+}
```

Comment:
We need to be careful here. If you look at the mmap implementation, technically there is some support for MMAP granularity smaller than wasm page size granularity. This could lead to surprising behavior from using this (seemingly simple) function, and we should at least document that.

Long term we probably should decide if we want to support the mmap hack.